fl4health/parameter_exchange/parameter_selection_criteria.py

```diff
@@ -28,8 +77,8 @@ def select_layers_by_threshold(
     initial_model_states = initial_model.state_dict()
     model_states = model.state_dict()
     for layer_name, layer_param in model_states.items():
-        layer_param_past = initial_model_states[layer_name]
-        drift_norm = _calculate_drift_norm(layer_param, layer_param_past, normalize)
+        ghost_of_layer_params_past = initial_model_states[layer_name]
```

Comment:
LOL. I really hope someone finds this in the source code some day and laughs a little bit.